src/lib/classes/notification.js

```diff
@@ -461,7 +461,7 @@ export class Notification {
          * Validate the list of notification IRIs and return a list of valid ones
          */
         const coreNotificationShape =
-          'https://shexshapes.inrupt.net/public/notifications/core-notification.shex';
+          'https://shexshapes.inrupt.net/public/notifications/app-notification.shex';
```

Comment:
Hhhmmm... I'm not sure - I'll have to look into that (it may have been a global search-and-replace mistake, as I don't remember needing to make a change there!). Cheers.